src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java

```diff
@@ -79,7 +79,7 @@ public AboutToStartOrSubmitCallbackResponse populatePreviewOrderWhenOrderUploade
 
 
 
-    @PostMapping(path = "/fetch-child-details", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
+    @PostMapping(path = "/fetch-order-details", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
```

Comment:
I found below reference to this call back old name in prl-ccd-definition in your branch, please see if that need update too.
"CallBackURLMidEvent": "${CCD_DEF_COS_URL}/fetch-child-details",

Reply:
Update CCD def (good spot!)